nextstrain/cli/runner/conda.py

```diff
@@ -471,8 +468,6 @@ def supported_os() -> bool:
         ('operating system is supported',
             supported_os()),
 
-        *test_rosetta_enabled(),
```

Comment:
Can the definition of this function be removed then?

Reply:
It's still used (optionally) for Docker:

https://github.com/nextstrain/cli/blob/4a00d7100eff811eab6df34db73c7f6d4196e22b/nextstrain/cli/runner/docker.py#L395